fastlane/lib/fastlane/actions/testfairy.rb

```diff
@@ -18,7 +18,12 @@ def self.run(params)
         return params[:ipa] if Helper.test?
 
         client_options = params.values
-        client_options[:testers_groups] = params[:testers_groups].join(',')
+
+        client_options[:testers_groups] = client_options[:testers_groups].join(',') if client_options.key?(:testers_groups)
+        client_options[:metrics] = client_options[:metrics].join(',') if client_options.key?(:metrics)
+        client_options[:options] = client_options[:options].join(',') if client_options.key?(:options)
+        client_options['auto-update'] = client_options.delete(:auto_update) if client_options.key?(:auto_update)
```

Comment:
Because (unfortunately) the keys that get passed need to be e.g. `auto-update` and not `auto_update`.